docs/MSPX.md

```diff
@@ -0,0 +1,18 @@
+# mLRS Documentation: MspX #
+
+([back to main page](../README.md))
+
+**Important: MSP support is experimental and currently only available in the branch 'dev-mspx'.**
+
+MspX is a technology designed to improve the over-the-air communication for systems using the MSP protocol, such as INAV systems. It introduces more robust framing and parsing which reduces packet losses to the minimum, and adds features such as compression of some very large MSP messages. It also converts MSP messages into CRSF telemetry messages to the radio, thus providng telemetry to the radio and Lua scripts running on the radio, such as the INAV app.
```

Comment:
Perhaps:

MspX is a technology designed to improve the over-the-air communication for systems using the MSP protocol, such as INAV. It includes the following features:
- Robust framing and parsing which reduces packet losses to a minimum.
- Compression of some very large MSP messages to reduce link utilization.
- MSP to CRSF message conversion which provides telemetry elements to the radio and therefore enables Lua scripts on the radio such as the INAV app to function correctly.

Reply:
THX!!!